validator_client/src/config.rs

```diff
@@ -222,6 +222,9 @@ impl Config {
             config.beacon_nodes_tls_certs = Some(tls_certs.split(',').map(PathBuf::from).collect());
         }
 
+        if cli_args.is_present("disable-run-on-all") {
+            config.broadcast_topics = vec![];
```

Comment:
Lets put a deprecation warning log here too

Reply:
Like `warn!(log, "DEPRECATED...")`?

Reply:
Yeah that's perfect